rpm/xrootd-hdfs.spec

```diff
@@ -12,7 +12,7 @@ Source0: %{name}-%{version}.tar.gz
 BuildRequires: xrootd-devel >= 1:4.6
 BuildRequires: xrootd-server-devel >= 1:4.6
 BuildRequires: cmake
-BuildRequires: hadoop-libhdfs-devel >= 2.0.0+545-1.cdh4.1.1
+BuildRequires: /usr/include/hdfs.h
```

Comment:
Ah, that's a bummer that hadoop 2.0.0 doesn't have a separate libhdfs-devel package.

Just a note here, apparently we used to care about the version requirement, but apparently now `hdfs.h` is only provided by versions of hadoop we still use, so it should be fine now as-is without the version requirement.